apps/trading/pages/index.page.tsx

```diff
@@ -8,7 +9,10 @@ export function Index() {
   const { replace } = useRouter();
   // The default market selected in the platform behind the overlay
   // should be the oldest market that is currently trading in continuous mode(i.e. not in auction).
-  const { data, error, loading } = useMarketList();
+  const { data, error, loading } = useDataProvider({
+    dataProvider: activeMarketsDataProvider,
+    update: () => true,
```

Comment:
question (non blocking): Does it make sense to provide a sensible default update function so this doesn't have to be passed in everywhere?